docs/product-user-guide/src/main/asciidoc/decision-tables-defining-proc.adoc

```diff
@@ -0,0 +1,44 @@
+[id='decision-tables-defining-proc']
+= Defining decision tables
+
+Decision table spreadsheets (XLS or XLSX) require two key areas that define rule data: a `RuleSet` area and a `RuleTable` area. The `RuleSet` area of the spreadsheet defines elements that you want to apply globally to all rule tables in the same package (not only the spreadsheet), such as a rule set name or universal rule attributes. The `RuleTable` area defines the actual rules (rows) and the conditions, actions, and other rule attributes (columns) that constitute that rule table within the specified rule set. A decision table spreadsheet can contain multiple `RuleTable` areas, but only one `RuleSet` area.
+
+Refer to the following sample spreadsheet as you define your decision table:
+
+.Sample decision table spreadsheet
+image::1248.jpg[]
+
+.Procedure
+. In a new XLS or XLSX spreadsheet, go to the second or third column and label a cell `RuleSet` (row 9 in example). Reserve the column or columns to the left for descriptive metadata (optional).
+. In the next cell to the right, enter a name for the `RuleSet`. This named rule set will contain all `RuleTable` rules defined in the rule package.
+. Under the `RuleSet` cell, define any rule attributes (one per cell) that you want to apply globally to all rule tables in the package. Specify attribute values in the cells to the right. For supported attributes, see xref:decision-tables-rule-set-entries-ref[].
+. Below the the `RuleSet` area and in the same column as the `RuleSet` cell, skip a row and label a new cell `RuleTable` (row 13 in example) and enter a table name in the same cell. The name string is used as the initial part of the name for all rules derived from this rule table, with the row number appended for distinction. You can override this automatic naming by inserting a `NAME` attribute column.
+. Use the next four rows to define the following elements as needed (rows 14-17 in example):
++
+*<@Michael (1 of 6): I'm not clear here on which cells are required and which are optional. The drools doc says that really just the rule attribute (first bullet below) is required, like CONDITION, etc., and the others are optional. But the user needs to specify at least the second cell too, the object type, right? If all are optional, then how would the user specify the condition or action or whatever? It would just say CONDITION and then the value of the first data cell, 3 rows down, would have what? Maybe a complete standalone code snippet or something? Can you give an example of a table that leaves all the optional cells blank?>*
```

Comment:
@sterobin You are pushing my knowledge! I however checked some Unit Tests we have and found an XLS where a CONDITION column does not have a OBJECT TYPE cell populated but has a full expression, including an object type, in the CONSTRAINT cell (e.g. the constraint value was ```Person( age == $1 )```). If a CONDITION column has ALL the other rows (OBJECT_TYPE and CONSTRAINT) empty but the the column has data for interpolation then compiling of the XLS will fail. So although the documents state OBJECT_TYPE and CONSTRAINT are optional in reality you need to provide one or the other or both.